.github/workflows/ci.yml

```diff
@@ -1,40 +1,69 @@
-name: ci
+name: CI
 
 on:
   push:
-    branches:
-      - develop
-      - dependabot/develop
+    branches: [main, develop, dependabot/develop]
   pull_request:
-    branches:
-      - develop
-      - dependabot/develop
+    branches: [main, develop, dependabot/develop]
+
+env:
+  SCHEMA_VERSION: "${{ github.ref == 'refs/heads/main' && 'main' || 'dev' }}"
```

Comment:
Worth leaving a comment here that this is `dev` because that's the name of the branch on the API repo, or people might get confused down the line as to `develop`/`dev` differences!